src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/BedSearchServiceTest.kt

```diff
@@ -533,20 +533,22 @@ class BedSearchServiceTest {
             name = "Room Characteristic Name",
           ),
         ),
+        overlaps = mutableListOf(),
       ),
     )
 
     every {
       mockBedSearchRepository.findTemporaryAccommodationBeds(
         startDate = LocalDate.parse("2023-03-22"),
-        durationInDays = 7,
```

Comment:
I'd expect to see this change in [the earlier commit which changes the parameter name](https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/908/commits/13f4933598eb954f5315662652cd7a7305450896). Presumably the tests would fail on that previous commit. Perhaps these two commits could be squashed? https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/908/commits/13f4933598eb954f5315662652cd7a7305450896 + https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/908/commits/0fd2cf49ca919b109d7156204ba5bd4f1a6cc783

Reply:
I've done some significant squashing and re-ordering to attempt to reconcile this issue.